detection_rules/beats.py

```diff
@@ -4,14 +4,18 @@
 
 """ECS Schemas management."""
 import os
+from pathlib import Path
 
 import kql
 import eql
 import requests
 import yaml
 
 from .semver import Version
-from .utils import unzip, load_etc_dump, save_etc_dump, get_etc_path, cached
+from .utils import unzip, load_dump, save_etc_dump, cached
+
+ROOT_DIR = Path(__file__).parent.parent
+ETC_DIR = ROOT_DIR.joinpath('etc')
```

Comment:
```suggestion
ETC_DIR = ROOT_DIR / 'etc'
```

Reply:
I'm happy to use `ETC_DIR = ROOT_DIR / 'etc'` instead of `ROOT_DIR.joinpath('etc')`, but the reason I started using `.joinpath()` is because I thought it would help with readability. I'm open to either style.

Reply:
If it's obvious to the team that something like `ETC_DIR / "beats_schemas" / latest` is joining paths, then I can leave out the calls to `joinpath()`

Reply:
Opted to use `/` to join paths where possible